app/views/generic_files/_embargo_help.html.erb

```diff
@@ -0,0 +1,7 @@
+<p>Embargo description goes here...</p>
```

Comment:
@straleyb I think we don't need this form any more right? I had added this partial form for the help bubble. Since we agreed that the embargo option would go under visibility, we could move this form and merge it with https://github.com/projecthydra/sufia/blob/master/app/views/generic_files/_visibility.html.erb